content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst

```diff
@@ -0,0 +1,76 @@
+===================================================================
+Choosing the right inventory flow to handle receipts and deliveries
+===================================================================
+
+There are several ways to handle how a warehouse receives products (receipts) and ships products
+(deliveries). Depending on factors such as the type of products stocked and sold, warehouse size,
+and the amount of daily confirmed receipts and delivery orders, the way products are handled when
+coming in and out of the warehouse can vary a lot. Different settings can be configured for receipts
+and deliveries; they do not have to be configured to have the same number of steps.
+
+Odoo handles shipping and receiving in 3 different ways, in one, two, or three steps. The simplest
+configuration is one step, which is the default. Each additional step required for a warehouse for
+either the receiving or shipping process will add an additional layer of operations to perform
+before a product is either received or shipped. These configurations depend entirely on the
+requirements for the products stored, such as performing quality checks on received products, or
+using special packaging on shipped products.
+
+One step flow
+=============
+
+**Receipt**: Receive products directly into stock. No intermediate steps between receipt and stock
+occur, such as a transfer to a quality location.
+
+**Shipping**: Ship products directly from stock.  No intermediate steps between stock and shipping
+occur, such as a transfer to a packing location.
+
+- Can only be used if not using :abbr:`FIFO (First In, First Out), :abbr:`LIFO (Last In, First
```

Comment:
Looks like I am missing a "`"  here:
 :abbr:`FIFO (First In, First Out)`,
 
 The FIFO is not showing up on the html doc, and also, i don't see the comma between the two listed and there should be commas between all of them.